apps/wpcom-block-editor/package.json

```diff
@@ -19,7 +19,7 @@
 		"bundle": "BROWSERSLIST_ENV=wpcom calypso-build --env WP",
 		"build:dev": "yarn run bundle",
 		"build:prod": "NODE_ENV=production yarn run bundle",
-		"build": "yarn clean && npm-run-all --parallel 'build:*'",
+		"build": "yarn clean && npm-run-all --parallel \"build:* {@}\" --",
```

Comment:
Note this change is temporary in this PR as the issue it resolves is being fixed elsewhere.  Updating this was required to run the dev build command in the meantime.